ddocs/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "eslint:recommended",
+  "env": {
+    "couchdb/couchdb": true,
+  },
+  "plugins": ["couchdb"],
+  "parserOptions": {
```

Comment:
We don't need parserOptions here, do we? Does it make any difference? Essential all the couchdb scripts i.e. _map.js, reduce.js, validate_doc_update.js_ are already parsed by the plugin

Reply:
It does make a difference, yeah. Specifically it reduces the compatability to ES5, which is important for CouchDB.

I considered hard-coding this in the plugin, but there may be people who only actually use PouchDB; or for which there are views they wish to lint that are only used in PouchDB. It also means it will support CouchDB's eventual JS library update (because only you know if you're deploying to the later version that supports ES6).

This is what it looks like with that parser option:
![image](https://user-images.githubusercontent.com/583851/51969633-dfb8aa00-246c-11e9-82ba-54103f10611f.png)


If I take that away you are allowed to use `const` etc again.

Reply:
Makes sense